tests/Unit/Evolution/Systems/Cce/Actions/Test_Psi0Matching.cpp

```diff
@@ -0,0 +1,277 @@
+          make_not_null(&coefficient_distribution), 1, l_max);
+      Spectral::Swsh::inverse_swsh_transform(
+          l_max, 1, make_not_null(&generated_data), generated_modes);
+      // aggressive filter to make the uniformly generated random modes
+      // somewhat reasonable
+      Spectral::Swsh::filter_swsh_boundary_quantity(
+          make_not_null(&generated_data), l_max, l_max / 2);
+      tensor_component = generated_data.data();
+    }
+  });
+
+  // (b) Puts those variables in two places: the
+  // MockRuntimeSystem runner and a databox called expected_box.
+  ActionTesting::emplace_component_and_initialize<component>(
+      &runner, 0,
+      {real_variables, swsh_volume_variables, swsh_boundary_variables,
+       Spectral::Swsh::SwshInterpolator{}});
+  auto expected_box = db::create<
+      tmpl::append<component::simple_tags, db::AddSimpleTags<Tags::LMax>>>(
+      std::move(real_variables), std::move(swsh_volume_variables),
+      std::move(swsh_boundary_variables), Spectral::Swsh::SwshInterpolator{},
+      l_max);
+
+  // (c) Calls next_action twice (which fills the results in the databox of
+  // MockRuntimeSystem runner).
+  runner.set_phase(Parallel::Phase::Evolve);
+  // Obtain inertial angular coordinates and SwshInterpolator that are needed
+  // by `CalculatePsi0AndDerivAtInnerBoundary`. This action is tested in
+  // Test_UpdateGauge.cpp
+  ActionTesting::next_action<component>(make_not_null(&runner), 0);
+  // Apply the `CalculatePsi0AndDerivAtInnerBoundary` action
+  ActionTesting::next_action<component>(make_not_null(&runner), 0);
+
+  // (d) Calls all the individual mutators by hand, filling the results in
+  // expected_box.
+  db::mutate_apply<GaugeUpdateAngularFromCartesian<
+      Tags::PartiallyFlatAngularCoords, Tags::PartiallyFlatCartesianCoords>>(
+      make_not_null(&expected_box));
+  db::mutate_apply<GaugeUpdateInterpolator<Tags::PartiallyFlatAngularCoords>>(
+      make_not_null(&expected_box));
+
+  db::mutate_apply<TransformBondiJToCauchyCoords>(make_not_null(&expected_box));
+  db::mutate_apply<PreSwshDerivatives<Tags::Dy<Tags::BondiJCauchyView>>>(
+      make_not_null(&expected_box));
+  db::mutate_apply<
+      PreSwshDerivatives<Tags::Dy<Tags::Dy<Tags::BondiJCauchyView>>>>(
+      make_not_null(&expected_box));
+  db::mutate_apply<VolumeWeyl<Tags::Psi0Match>>(make_not_null(&expected_box));
+  db::mutate_apply<PreSwshDerivatives<Tags::Dy<Tags::Psi0Match>>>(
+      make_not_null(&expected_box));
+  db::mutate_apply<InnerBoundaryWeyl>(make_not_null(&expected_box));
+
+  // (e) Compares MockRuntimeSystem's databox vs expected_box.
+  tmpl::for_each<swsh_volume_tags_to_compute>(
+      [&runner, &expected_box](auto tag_v) {
+        using tag = typename decltype(tag_v)::type;
+        const auto& test_lhs =
+            ActionTesting::get_databox_tag<component, tag>(runner, 0);
+        const auto& test_rhs = db::get<tag>(expected_box);
```

Comment:
I find `test_lhs` and `test_rhs` to be not very descriptive names. What about `expected_result` and `action_result`? Same on line 274 below